bears/shell/BashateBear.py

```diff
@@ -0,0 +1,32 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='bashate',
+        output_format='regex',
+        output_regex=r'\[.*\] (?P<origin>E\d+): '
+                     r'(?P<message>.+):(?:.*\s.*): '
```

Comment:
unnecessary use of anonymous group: -->
```
.*\s.*
```

Reply:
What's actually put there as message from the tool?

Reply:
the file location

Reply:
can a linter bear be passed multiples files at once?